scripts/installscripts/install-jukebox.sh

```diff
@@ -1138,9 +1130,6 @@ install_main() {
         echo "classic" > "${jukebox_dir}"/settings/edition
     fi
 
-    # update mpc / mpd DB
```

Comment:
We used to do this to make sure the DB is up to date?

Reply:
It looks like you're also using Mopidy-MPD instead of mpd when in Spotify enabled mode? Somewhat shockingly, I don't believe `mpc rescan` will work for Mopidy-MPD since we (still!) don't implement MPD's `update` command, it'll just ignore it. The only way to trigger a database update for the service is via `sudo mopidyctl local scan`.

Reply:
Hmm good point. I will check that and add the case.

Reply:
And full disclosure, you might already be aware, but Mopidy-MPD is still definitely lacking compared to regular mpd. We have fallen way behind in updating the protocol and some clients have issues with this, either providing less functionality or not working at all. It's always been a goal to close the gap but it hasn't happened yet. I don't use MPD clients myself anymore so I'm not personally motivated enough. If any of your users would be motivated to help improve this, that would be really great.

Saying that, I might try and fix the update command because it's a bit mad that's still not working.